content/00_Foreword.md

```diff
@@ -1,185 +1,105 @@
-
-The version that this book has been written against is the .Net 4.0 targeted Rx
-assemblies version 1.0.10621.0 (NuGet: Rx-Main v1.0.11226).
-
-So, fire up Visual Studio and let's get started.
-
----
-
-<div class="webonly">
-	<h1 class="ignoreToc">Additional recommended reading</h1>
-	<div align="center">
-		<div style="display:inline-block; vertical-align: top;  margin: 10px; width: 140px; font-size: 11px; text-align: center">
-			<!--C# in a nutshell Amazon.co.uk-->
-			<iframe src="http://rcm-uk.amazon.co.uk/e/cm?t=int0b-21&amp;o=2&amp;p=8&amp;l=as1&amp;asins=B008E6I1K8&amp;ref=qf_sp_asin_til&amp;fc1=000000&amp;IS2=1&amp;lt1=_blank&amp;m=amazon&amp;lc1=0000FF&amp;bc1=000000&amp;bg1=FFFFFF&amp;f=ifr" 
-					style="width:120px;height:240px;margin: 10px" 
-					scrolling="no" marginwidth="0" marginheight="0" frameborder="0"></iframe>
-
-		</div>
-		<div style="display:inline-block; vertical-align: top;  margin: 10px; width: 140px; font-size: 11px; text-align: center">
-			<!--C# Linq pocket reference Amazon.co.uk-->
-			<iframe src="http://rcm-uk.amazon.co.uk/e/cm?t=int0b-21&amp;o=2&amp;p=8&amp;l=as1&amp;asins=0596519249&amp;ref=qf_sp_asin_til&amp;fc1=000000&amp;IS2=1&amp;lt1=_blank&amp;m=amazon&amp;lc1=0000FF&amp;bc1=000000&amp;bg1=FFFFFF&amp;f=ifr" 
-					style="width:120px;height:240px;margin: 10px" 
-					scrolling="no" marginwidth="0" marginheight="0" frameborder="0"></iframe>
-		</div>
-
-		<div style="display:inline-block; vertical-align: top; margin: 10px; width: 140px; font-size: 11px; text-align: center">
-			<!--CLR via C# v4 Amazon.co.uk-->
-			<iframe src="http://rcm-uk.amazon.co.uk/e/cm?t=int0b-21&amp;o=2&amp;p=8&amp;l=as1&amp;asins=B00AA36R4U&amp;ref=qf_sp_asin_til&amp;fc1=000000&amp;IS2=1&amp;lt1=_blank&amp;m=amazon&amp;lc1=0000FF&amp;bc1=000000&amp;bg1=FFFFFF&amp;f=ifr" 
-					style="width:120px;height:240px;margin: 10px" 
-					scrolling="no" marginwidth="0" marginheight="0" frameborder="0"></iframe>
-
-		</div>
-		<div style="display:inline-block; vertical-align: top; margin: 10px; width: 140px; font-size: 11px; text-align: center">
-			<!--Real-world functional programming Amazon.co.uk-->
-			<iframe src="http://rcm-uk.amazon.co.uk/e/cm?t=int0b-21&amp;o=2&amp;p=8&amp;l=as1&amp;asins=1933988924&amp;ref=qf_sp_asin_til&amp;fc1=000000&amp;IS2=1&amp;lt1=_blank&amp;m=amazon&amp;lc1=0000FF&amp;bc1=000000&amp;bg1=FFFFFF&amp;f=ifr" 
-					style="width:120px;height:240px;margin: 10px" 
-					scrolling="no" marginwidth="0" marginheight="0" frameborder="0"></iframe>
-
-		</div>           
-	</div>
</div>
-
+---
+title : Foreword
+---
+
+# Introduction to Rx
+By Ian Griffiths and Lee Campbell
+   
+---
+
+Reactive programming is not a new concept. Any kind of user interface development
+necessary involves code that responds to events. Languages like Smalltalk, Delphi
+and the .NET languages have popularized reactive or event-driven programming paradigms.
+Architectural patterns such as CEP (Complex Event Processing), and
```

Comment:
Do we need links out to some suitable definition for CEP, CQRS or even Smalltalk and Delphi in the previous paragrpah? Antiques like us might know them, but does everyone?